testflinger_agent/client.py

```diff
@@ -267,10 +292,42 @@ def post_agent_data(self, data):
         :param data:
             dict of various agent data points to send to the api server
         """
-        agent = self.config.get("agent_id")
         agent_data_uri = urljoin(self.server, "/v1/agents/data/")
-        agent_data_url = urljoin(agent_data_uri, agent)
+        agent_data_url = urljoin(agent_data_uri, self.config.get("agent_id"))
         try:
             self.session.post(agent_data_url, json=data, timeout=30)
         except RequestException as exc:
             logger.error(exc)
+
+    def post_influx(self, job_id, phase, duration, result):
+        """Post the relevant data points to testflinger server
+
+        :param data:
+            dict of various agent data points to send to the api server
+        """
+        measurement = "%s phase result" % job_id
```

Comment:
DEFINITELY don't encode job_id in measurement. Let's just drop this job_id from anything here unless we have a clear reason for it. If I'm wrong, please tell me! but I can't currently think of any reason why we want the job_id in these results.

Reply:
The only thought I had was it might be useful to have correlated data against the job id. I just don't want it to feel left out.

Seriously though, I'll get rid of it next push.

Reply:
encoding it into the measurement though would mean you would have a different measurement for every single job ID. I'm fairly sure that this is the field you need to manually select in grafana to tell it what you want to run the query against. So we definitely want this to be the same string for all of these measurements, or else it's going to be difficult to get results out of it.

Reply:
yeah, that makes sense. it will be clear once we start using grafana in depth here.